web-frontend/src/pages/Assets/DeleteAssets/DeleteAssetModal.tsx

```diff
@@ -0,0 +1,30 @@
+import { PopupModal, PopupModalHeader } from '@massalabs/react-ui-kit';
+
+import Intl from '@/i18n/i18n';
+import { ConfirmDelete } from '@/pages/Assets/DeleteAssets/ConfirmDelete';
+
+interface DeleteAssetModal {
+  setModalOpen: (value: boolean) => void;
+  tokenAddress: string;
+}
+
+export function DeleteAssetModal({ ...props }: DeleteAssetModal) {
+  const { setModalOpen } = props;
+
+  return (
+    <PopupModal
+      customClass="!w-[440px] h-1/2"
+      fullMode={true}
+      onClose={() => setModalOpen(false)}
```

Comment:
You can try:

```suggestion
      {...props}
```
------

**OR**

------

```suggestion
      onClose={() => onClose()}
```

@mazmassa What do you think? 🤔

Reply:
loool check this out : 
<img width="341" alt="Screenshot 2023-08-10 at 15 33 43" src="https://github.com/massalabs/station-massa-wallet/assets/90157528/d8d07141-e3f2-4202-b67d-296b4008dd69">

Reply:
I'm gonna name it closeModal

Reply:
plus I'm not a big fan of such abstraction, code is less readable imho

Reply:
implemented like this :  ```onClose={() => closeModal()}```